pepagent/pepagent.py

```diff
@@ -468,3 +624,51 @@ def _check_conn_db(self) -> None:
         cols_name.sort()
         if DB_COLUMNS != cols_name:
             raise SchemaError
+
+
+def main():
+    # Create connection to db:
+    # projectDB = PepAgent(
+    #     user="postgres",
+    #     password="docker",
+    # )
+    projectDB = PepAgent("postgresql://postgres:docker@localhost:5432/pep-base-sql")
+
+    # prp_project2 = peppy.Project("/home/bnt4me/Virginia/pephub_db/sample_pep/amendments2/project_config.yaml")
+    # projectDB.upload_project(prp_project2, namespace="King", anno={"sample_anno": "Tony Stark "})
+
+    # Add new projects to database
+    # directory = "/home/bnt4me/Virginia/pephub_db/sample_pep/"
+    # os.walk(directory)
+    # projects = (
+    #     [os.path.join(x[0], "project_config.yaml") for x in os.walk(directory)]
+    # )[1:]
+    #
+    # print(projects)
+    # for d in projects:
+    #     try:
+    #         prp_project2 = peppy.Project(d)
+    #         projectDB.upload_project(prp_project2, namespace="King", tag="new_tag", anno={"sample_anno": "Tony Stark "})
+    #     except Exception:
+    #         pass
+
+    # dfd = projectDB.get_project(registry="King/amendments2")
+    # print(dfd)
+    # dfd = projectDB.get_projects(tag="new_tag")
+    # print(dfd)
+    # dfd = projectDB.get_namespaces()
+    # print(dfd)
+    # dfd = projectDB.get_namespace(namespace="other")
+    # print(dfd)
+
+    d = projectDB.get_namespace_annotation()
+    print(d)
+    # print(projectDB.get_namespace_annotation())
+
+
+if __name__ == "__main__":
+    try:
+        sys.exit(main())
+    except KeyboardInterrupt:
+        print("Pipeline aborted.")
+        sys.exit(1)
```

Comment:
These should just be unit tests. I don't think we need a `main()` function in a python package that isn't intended to have `cli` functionality. Sort of like [`peppy`](https://github.com/pepkit/peppy/blob/5b519de4f5ca4c5b598f0af7d06cfb7178bb04ee/peppy/project.py#L1180)

Reply:
I agree with that. I added this lines for local testing. After we will have some stable version I will delete it.

Reply:
YEAH, and I should add some new unit tests...